### PR TITLE
New rules for Composite:GPSDateTime tag and filesystem created time

### PR DESCRIPTION
**Pull request by pivis**
Adding a default rule for Composite:GPSDateTime + gps timezone (looks like it's UTC) + a few rules that are more controversial to the predefined rules list (filesystem created time + Composite:GPSDateTime in UTC and in user default timezone).

### CODE REVIEW

**Comment by pivis** [comment]
While in many cases filesystem created time and modified time are same, they also can be different + if someone specifically maintains their library to have one of these as "correct" then it makes sense to let people choose it specifically, so I'm adding created time in addition to modified time to fully address request in https://github.com/LibrePhotos/librephotos/issues/388 .

The rules with Composite:GPSDatetime are the ones for https://github.com/LibrePhotos/librephotos/issues/300 .

File: api/exif_tags.py [hunk 1]
```diff
@@ -6,3 +6,13 @@ class Tags:
...
+    GPS_DATE_TIME = "Composite:GPSDateTime"
+    RIFF_DATE_TIME_ORIGINAL = "RIFF:DateTimeOriginal"
+    MAKERNOTES_DATE_TIME_ORIGINAL = "MakerNotes:DateTimeOriginal"
+    MAKERNOTES_MAKE = "MakerNotes:Make"
+    EXIF_MAKE = "EXIF:Make"
+    MAKERNOTES_MODEL = "MakerNotes:Model"
+    EXIF_MODEL = "EXIF:Model"
+    MAKERNOTES_TYPE = "MakerNotes:MakerNoteType"
+    MAKERNOTES_VERSION = "MakerNotes:MakerNoteVersion"
+    EXIF_ModifyDate = "EXIF:ModifyDate"
```
Comment (derneuere) [inline api/exif_tags.py]: This tag looks different from the other ones 😅

**Comment by derneuere** [comment]
Thanks for the pull request! Looks good to me, only found the one small issue regarding the naming of a tag. Will merge it, when that get fixed 😄

**Comment by pivis** [comment]
Thank you for taking a look. Fixed the name casing so it looks like others. I'm not really using it (and majority of others added here), but some of these might be useful later. I think only `Composite:GPSDateTime` is really used here out of the new ones that are added.

**Comment by derneuere** [comment]
Alright good to know! Looks good to me 😄

**Pull request merged**